src/app/routes/utils/regex/utils/index.js

```diff
@@ -154,6 +154,11 @@ export const getTopicPageRegex = services => {
   return `/:service(${serviceRegex}):variant(${variantRegex})?/topics/:id(${topicIdRegex})?:lite(${liteRegex})?`;
```

Comment:
Should we consider removing variant from this regex now, given that it's already catered for in the new regex? Or is this to ensure backwards compatibility with the "old" topic routing?

Reply:
Are there plans to combine both regexes into a single one (in the new format?) e.g. once the Belfrage changes are made to redirect all variant topics to the new format?

Reply:
Yea we have 2 tickets to 'clean up' the old routes here in Simorgh and in Belfrage once this change and the Belfrage one are on Live.

Reply:
Great, thanks! 👐